From 987be669151005097c84d842563b45e0d04c9be5 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Mon, 23 Feb 2026 22:34:35 +0000
Subject: include/json_print.h: add includes for `__u32` and `timeval`
 declarations

Without the change the build fails on `musl` as:

    ../include/json_print.h:100:44: error: unknown type name '__u32'
---
 include/json_print.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/json_print.h b/include/json_print.h
index 59edd5b24..c0d6315f1 100644
--- a/include/json_print.h
+++ b/include/json_print.h
@@ -8,6 +8,10 @@
 #ifndef _JSON_PRINT_H_
 #define _JSON_PRINT_H_

+#include <sys/time.h> /* struct timeval typedef */
+
+#include <asm/types.h> /* __u32 typedef */
+
 #include "json_writer.h"
 #include "color.h"

